concordium-contracts-common/src/schema_json.rs

```diff
@@ -930,6 +932,41 @@ fn write_bytes_from_json_schema_type<W: Write>(
     }
 }
 
+impl Fields {
+    /// Displays a template of the JSON to be used for the Fields.
+    /// It should match the output of `concordium-client`.
```

Comment:
That's not really a requirement. The requirement is that it has to be as much as possible to copy the template and edit it to provide the actual parameters.

The comment about concordium-client does not really tell the caller of this function very much.